internal/cli/atlas/quickstart/prompt.go

```diff
@@ -16,28 +16,18 @@ package quickstart
 
 import (
 	"fmt"
-	"strconv"
-	"time"
 
 	"github.com/AlecAivazis/survey/v2"
-	"github.com/mongodb/mongocli/internal/randgen"
-	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/mongosh"
 	"github.com/mongodb/mongocli/internal/usage"
 	"github.com/mongodb/mongocli/internal/validate"
 )
 
-const (
-	mongoShellHelp         = "MongoDB CLI will use the MongoDB shell version provided to allow you to access your deployments."
-	mongoShellNotFoundHelp = "MongoDB CLI will store the path in your profile, type ‘mongocli config’ to change it."
-	SampleDataHelp         = "Add a sample dataset into your new Atlas cluster."
-	passwordLength         = 12
-)
-
-func newClusterNameQuestion(clusterName, message string) *survey.Question {
+func newClusterNameQuestion(clusterName string) *survey.Question {
 	return &survey.Question{
 		Name: "clusterName",
 		Prompt: &survey.Input{
-			Message: fmt.Sprintf("Cluster Name%s:", message),
+			Message: "Cluster Name [This can't be changed later]",
```

Comment:
I personally don't like this change, I think we can add this information to the `usage.ClusterName` and leave the "Press Enter message.." message. Was this requested by @viviandiancheng?

Reply:
This is from me after doing some fixes to config and seeing some of the good things we did there to nudge users

This is the current quickstart
```shell
mongocli atlas quickstart 

[Set up your Atlas cluster]
? Name of the cluster.
? Cluster Name [Press Enter to use the auto-generated name 'Quickstart-1622726262']: (Quickstart-1622726262)
```

This after the change

```bash
mongocli atlas quickstart
You are creating a new Atlas cluster and enabling access to it.

Press [Enter] to use the default values.

Enter [?] on any option to get help.

[Set up your Atlas cluster]
? Cluster Name [This can't be changed later] [? for help] (Quickstart-1622726359)
```

We now call at the start to press `enter` or `?` without over complicating the input message happy to hear back from @viviandiancheng and @melissamahoney-mongodb